pybossa/view/bloomberg.py

```diff
@@ -88,7 +90,7 @@ def handle_bloomberg_response():
                 user_data['name']        = attributes['username'][0]
                 user_data['password']    = generate_password()
                 user_data['admin']       = 'BSSO'
-                user_data['user_type']   = get_user_type(attributes.get('firmId', [None])[0])
+                user_data['user_type']   = app_settings.upref_mdata.firm_id_to_type_mapping.get(attributes.get('firmId', [None])[0])
```

Comment:
is it permissible to have users created w/ firm ids other than the ones configured? If not, this should raise an exception to fail user creation.

Reply:
I think we agreed that anyone with access to BSSO could get a default L4 account.

Reply:
I think its anyone under configured firm with bsso access would get account created. We can talk and discuss